packages/react-fresh/src/__tests__/__snapshots__/ReactFreshBabelPlugin-test.js.snap

```diff
@@ -1,5 +1,61 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`ReactFreshBabelPlugin generates signatures for function declarations calling hooks 1`] = `
+"
```

Comment:
you can use https://www.npmjs.com/package/jest-snapshot-serializer-raw to avoid the escaped `"` and extra wrapping `"`. We've done the same in Jest: https://github.com/facebook/jest/pull/7651

Makes them a bit easier to read, IMO 🙂